### ISSUE TITLE
When train VOC, it displayed  "Loss: nan" around 50 iters. i.e. "iter 50 || Loss: nan ||". Anyone came across this problem?

### ISSUE THREAD

**Issue by xiaozhi2015** [amdegroot/ssd.pytorch#118]

**Comment by xiaozhi2015** [comment]
The only place I changed is "batch_size = 8"

**Comment by yuxwind** [comment]
Thanks. But I'm still confused why it works?
